shelley/chain-and-ledger/formal-spec/address.tex

```diff
@@ -3,14 +3,16 @@ \section{Addresses}
 
 Addresses are described in section 4.2 of the delegation design document \cite{delegation_design}.
 The types needed for the addresses are defined in Figure~\ref{fig:defs:addresses}.
-There are three types of UTxO addresses:
+There are four types of UTxO addresses:
 \begin{itemize}
   \item Base addresses, $\AddrB$,
         containing the hash of a payment key and the hash of a staking key,
   \item Pointer addresses, $\AddrP$,
         containing the hash of a payment key and a pointer to a stake key registration certificate,
   \item Enterprise addresses, $\AddrE$,
         containing only the hash of a payment key (and which have no staking rights).
+  \item Bootstrap addresses, $\AddrBS$,
+        corresponding to the addresses in Byron, behaving exactly like enterprise addresses.
```

Comment:
should we give a citation for Byron?

Reply:
That's a good idea.

Reply:
When we do #396 it will be even more clear how this address corresponds to the address in Byron.